client/incident.go

```diff
@@ -163,3 +176,11 @@ type GetIncidentsResults struct {
 	HasMore    bool       `json:"has_more"`
 	Items      []Incident `json:"items"`
 }
+
+// StatusUpdateOptions are the fields required to update an incident's status
+type StatusUpdateOptions struct {
+	IncidentID        string `json:"incident_id"`
```

Comment:
This is not needed in the JSON body, right? As far as I understand, it is only used when building the URL. If so, we can ignore it when marshaling:
```suggestion
	IncidentID        string `json:"-"`
```

Reply:
I think this is for marshalling to /send/ to the server.

Reply:
Yes, but the server only reads the Message, Reminder and Status, right?

Reply:
Right -- I'm removing `incident_id` and also making the argument list change below. Thanks!

Reply:
Cool, thanks!